plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/AggregateMediatorExtFactory.java

```diff
@@ -96,13 +100,13 @@ protected Mediator createSpecificMediator(OMElement omElement) {
                 OMAttribute min = messageCount.getAttribute(MIN_Q);
                 if (min != null) {
                     ((AggregateMediator) mediator)
-                            .setMinMessagesToComplete(new ValueFactory().createValue("min", messageCount));
+                            .setMinMessagesToComplete(new ValueExtFactory().createValue("min", messageCount));
```

Comment:
What does Ext mean? Let's have rename the class to have the whole word since having enigmatic variable names is not prefered.

Reply:
Fixed ...